Dockerfile

```diff
@@ -12,7 +12,9 @@ COPY Pipfile* /app/
 RUN apt-get update \
     && apt-get install -y libpq-dev gcc \
     && rm -rf /var/lib/apt/lists/* \
-    && pip install --upgrade pip
+    && pip install --upgrade pip \
+    && pip install python-dotenv \
+    && pip install flask
```

Comment:
Could you explain the purpose of installing flask with pip and pipenv? Is there something I'm missing?

Reply:
I'm running this on Linux Mint 19 with docker and docker-compose already installed. When I ran the `make`, `make build` and `make setup` commands my bash interpreter recommended installing those packages. Once I added them to the Dockerfile the shell had no more complaints.